src/utilities/byte.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef SRC_UTILITIES_BYTE_HPP_
+#define SRC_UTILITIES_BYTE_HPP_
+
+#include "cstdint"
+
+namespace utilities {
+
+class Byte {
+ public:
+  ~Byte() = default;
+
+  Byte(){};
+  Byte(std::uint8_t byte) : byte_(byte){};
```

Comment:
Codacy found an issue: [Class 'Byte' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/SquidSquad/Squiddrone/pullRequest?prid=6098018)